python/lsst/ts/idl/enums/MTDome.py

```diff
@@ -80,6 +80,7 @@ class MotionState(enum.IntEnum):
     """
 
     ERROR = enum.auto()
+    DISABLED = enum.auto()
```

Comment:
This renumbers all the existing values below. Is that safe? Will there be existing EFD entries invalidated by that change? (In other words, should this new value be put at the end, instead of inserted here?)

Also: do you need to update an enum in ts_xml as well?

Reply:
Good points. I didn't think of those. I'll add it at the bottom instead and will check ts_xml.